src/Themes/Sample.Theme/Recipes/DotNest.Core.SDK.MediaTheme.recipe.json

```diff
@@ -1,50 +0,0 @@
-{
-  "name": "DotNest.Core.SDK.MediaTheme",
-  "displayName": "Media Theme",
-  "description": "A recipe created with the media-theme-deployment tool.",
-  "author": "Lombiq Technologies",
-  "website": "https://github.com/Lombiq/DotNest-Core-SDK",
-  "version": "",
-  "issetuprecipe": false,
-  "categories": [],
-  "tags": [],
-  "steps": [
-    {
-      "name": "Feature",
-      "enable": [
-        "Lombiq.Hosting.MediaTheme.Bridge",
-        "Lombiq.Hosting.MediaTheme"
-      ]
-    },
-    {
-      "name": "Themes",
-      "Site": "Lombiq.Hosting.MediaTheme"
-    },
-    {
-      "name": "mediatheme",
-      "ClearMediaThemeFolder": true,
-      "BaseThemeId": "TheTheme"
-    },
-    {
-      "name": "media",
-      "Files": [
-        {
-          "SourcePath": "_MediaTheme/Assets/example-local.png",
-          "TargetPath": "_MediaTheme/Assets/example-local.png"
-        },
-        {
-          "SourcePath": "_MediaTheme/Assets/example.png",
-          "TargetPath": "_MediaTheme/Assets/example.png"
-        },
-        {
-          "SourcePath": "_MediaTheme/Templates/Example.liquid",
-          "TargetPath": "_MediaTheme/Templates/Example.liquid"
-        },
-        {
-          "SourcePath": "_MediaTheme/Templates/Widget__LayoutInjection.liquid",
-          "TargetPath": "_MediaTheme/Templates/Widget__LayoutInjection.liquid"
-        }
-      ]
-    }
-  ]
```

Comment:
Don't we need these as examples for the media theme?

Reply:
We decided to delete it with Bence because this is a generated recipe, and none of the other themes has it committed.